core/src/main/resources/lib/form/secretTextarea.jelly

```diff
@@ -56,17 +56,32 @@ Example usage:
         <st:attribute name="placeholder">
             Placeholder text for input field when displayed.
         </st:attribute>
+        <st:attribute name="checkUrl">
```

Comment:
there is no `use=optional` here and this was copied from other jelly files.  placeholder also does not have it, so I can only conclude that it is unused?  
if this is used then I would suggest that it is fixed in all Jelly files at once rather than peicemeal which is not the realm of this PR.